workspace/spec/lita/handlers/teamcity_spec_anki.rb

```diff
@@ -93,4 +93,42 @@
     end
 
   end
+
+  describe '#list' do
+    build_type_json_str = '{"buildType":[{"id":"Wildcard_A","projectName":"Wildcard :: A"},
+                                         {"id":"B_Wildcard","projectName":"Wildcard :: B"}]}'
+    before do
+      stub_request(:get, /buildTypes/).to_return(:status => 200, :body => build_type_json_str)
+    end
+
+    it 'reply the builds if wildcard exists' do
```

Comment:
I think the description should be minor changed, e.g. 'replies the correct builds if wildcard exists'